packages/nx/src/utils/command-line-utils.ts

```diff
@@ -290,7 +290,7 @@ function getUntrackedFiles(): string[] {
   return parseGitOutput(`git ls-files --others --exclude-standard`);
 }
 
-function getMergeBase(base: string, head: string = 'HEAD') {
+function getMergeBase(base: string = 'main', head: string = 'HEAD') {
   try {
```

Comment:
Default to main

Reply:
This is not related to your PR please revert this